website/docs/components/form/primitives/partials/guidelines/guidelines.md

```diff
@@ -9,6 +9,51 @@
 
 - Use helper text to give the user extra details about the data you’re asking them to input, e.g., formatting requirements such as MM-DD-YYYY.
 
+### Form::CharacterCount
+
+Use a character count to communicate the current length of the value in an input and whether it meets or exceeds the length requirements passed to the component. The component accepts multiple arguments to set length requirements and exposes several computed values to support custom messages. Visit the [code](/components/form/primitives?tab=code#formcharactercount-1) documentation for more details. <!-- TODO: ensure that this link is correct -->
+
+#### Default messages
```

Comment:
I did mockup some videos of this but decided not to include them because the code docs outline all of the relevant examples. I can bring them back in though, the assets are more or less already created.

Reply:
I think our design audience will thank you 🙂

Reply:
Added an overview video in https://github.com/hashicorp/design-system/pull/1799/commits/bdb20bbaad868b50d771a4b2abe26be5f44ac676 capturing all of the messages in one (seemed like the best solution for brevity, but I can break it up into individual videos if we want.